docs/logs/getting-started/Program.cs

```diff
@@ -15,17 +15,40 @@
 // </copyright>
 
 using System.Collections.Generic;
+#if NETCOREAPP2_1
+using Microsoft.Extensions.DependencyInjection;
+#endif
 using Microsoft.Extensions.Logging;
+using OpenTelemetry;
 
 public class Program
 {
     public static void Main()
     {
+        /*
+        The OpenTelemetry style:
+
+        using var loggerProvider = Sdk.CreateLoggerProviderBuilder()
+            .AddProcessor(new MyProcessor())
+            .Build();
+        var logger = loggerProvider.CreateLogger("MyLogger");
+        */
+
+#if NETCOREAPP2_1
+        var serviceCollection = new ServiceCollection().AddLogging(builder =>
+#else
         using var loggerFactory = LoggerFactory.Create(builder =>
+#endif
         {
-            builder.AddOpenTelemetry();
+            builder.AddOpenTelemetry(options => options.AddProcessor(new MyProcessor()));
```

Comment:
Duplicate `builder.AddOpenTelemetry()` to the above `#if/#else/#endif`, so we don't need another `#if/#else/#endif` below might make the code more friendly to new comer?

Reply:
As I've explained in https://github.com/open-telemetry/opentelemetry-dotnet/pull/1315#issue-495159429, have a simple tutorial/demo is not in this PR's scope.